### DISCUSSION TITLE
Current Issue list (30 Sep 2024)

### DISCUSSION THREAD

**Discussion by ChrisMaunder** [codeproject/CodeProject.AI-Server#133] [Announcements]
There is, obviously, a bit of a backlog with issues (as of 30 Sep 2024). We will try and triage these as fast as possible over the next week or so, so please be patient.

**Comment by SlothCroissant** [comment]
Appreciate you, @ChrisMaunder. Looking forward to seeing things continue onward.

**Reply by ChrisMaunder** [reply]
@SlothCroissant Best. Username. Ever.

**Comment by ChrisMaunder** [comment]
2.9.5 is out and issues are now being assigned to the core server or the relevant module. Closing this discussion
